<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_changelog</class>
 <widget class="QWidget" name="form_changelog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lbl_cl_version">
         <property name="text">
          <string>Version:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ln_cl_version">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_status">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ln_status"/>
       </item>
       <item>
        <widget class="QLabel" name="lbl_urgency">
         <property name="text">
          <string>Urgency Level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_urgency">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>low</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>medium</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>high</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="txt_changelog"/>
     </item>
     <item row="4" column="0">
      <widget class="QListWidget" name="lsw_changelog">
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_changelog_add">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="insert-table-row">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_changelog_remove">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="delete-table-row">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QPushButton" name="btn_changelog_save">
       <property name="text">
        <string>Create changelog</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
